src/Sidekick.Common/Settings/Filter.cs

```diff
@@ -0,0 +1,29 @@
+namespace Sidekick.Common.Settings;
+public class Filter
+{
+    public FilterType FilterType { get; set; }
+    public string? Icon { get; set; }
+}
+
+public class Filters
+{
+    public static List<Filter> All =>
```

Comment:
I think this static value could be inside the component itself as it is used only once. No need to create a class in the Common project that isn't used in a Common way.